src/main/java/com/synopsys/integration/alert/provider/polaris/PolarisProperties.java

```diff
@@ -94,7 +100,12 @@ public AccessTokenRestConnection createRestConnection(final IntLogger intLogger)
 
     public AccessTokenRestConnection createRestConnection(final IntLogger intLogger, final String baseUrl, final String accessToken, final Integer timeout) {
         final Boolean alwaysTrustCertificate = alertProperties.getAlertTrustCertificate().orElse(Boolean.FALSE);
-        final ProxyInfo proxyInfo = alertProperties.createProxyInfo();
+        ProxyInfo proxyInfo = ProxyInfo.NO_PROXY_INFO;
+        try {
+            proxyInfo = proxyManager.createProxyInfo();
+        } catch (AlertDatabaseConstraintException | AlertRuntimeException ex) {
+            logger.error("Error creating proxy information", ex);
+        }
```

Comment:
Since this seems to be in multiple locations, maybe we can create a restconnection wrapper for everything that automatically adds our Proxy stuff to that connection.